### DISCUSSION TITLE
Ability to download/export pointclouds

### DISCUSSION THREAD

**Discussion by srikanthmalla** [foxglove/community#323] [Q&A]
Hi,
The following features would be nice to add to the current foxglove studio
1. Able to download pointclouds (pcd, txt, csv formats) at any time instance.
2. Allowing the poincloud to be downloaded w.r.t different transform (tf) from the transform tree. Since the current visualizer in foxglove already does do that for visualization.

Thank you so much,
Srikanth

**Comment by amacneil** [comment]
Thanks! No ETA on this but we agree it's a good feature to add.
